src/lib/audio/audio-util.js

```diff
@@ -0,0 +1,18 @@
+class AudioUtil {
+    static computeRMS (samples) {
```

Comment:
Now that we're using `import/export` syntax, we could just export this by itself.